### DISCUSSION TITLE
Is it posible connect 2 ESP32 as master/slave using BLEMIDI ?

### DISCUSSION THREAD

**Discussion by henriquegmatos** [lathoub/Arduino-BLE-MIDI#46] [Q&A]
It is possible to connect 2 ESP32 using BLEMIDI.

I've already managed to make a MIDI/BLE interface using ESP32 and connecting to an iphone. However I would like to connect ESP32 to another ESP32 to transport MIDI protocol using BLEMIDI. It is possible? Which BLEMIDI Library command is used to connect to a BLE-ID of another ESP32?

best regards

**Comment by lathoub** [comment]
Yes, that should be possible - on 1 ESP32 use this [example](https://github.com/lathoub/Arduino-BLE-MIDI/blob/master/examples/MidiBle/MidiBle.ino), and the other using the [ble-client](https://github.com/lathoub/Arduino-BLE-MIDI/blob/master/examples/MidiBle_Client/MidiBle_Client.ino) example.\

The client example is currently [being worked on](https://github.com/lathoub/Arduino-BLE-MIDI/issues/40), you can participate in the testing.

**Reply by jhsa** [reply]
> Yes, that should be possible - on 1 ESP32 use this [example](https://github.com/lathoub/Arduino-BLE-MIDI/blob/master/examples/MidiBle/MidiBle.ino), and the other using the [ble-client](https://github.com/lathoub/Arduino-BLE-MIDI/blob/master/examples/MidiBle_Client/MidiBle_Client.ino) example.\
> 
> The client example is currently [being worked on](https://github.com/lathoub/Arduino-BLE-MIDI/issues/40), you can participate in the testing.

Late for the party here ;) but is there a way of selecting if the ESP32 starts as a server or a client at power on? 
As far as my little knowledge goes I only found how to compile it as either a server or a client. I would like to have both available and decide which one to select at power ON.
Thanks
